### PR TITLE
Installed w-express-js and configured the project

### PR DESCRIPTION
**Pull request by minotad**
[pull_request_template.md](https://github.com/wolox-training/dmq-express-js/files/6533590/pull_request_template.md)

### CODE REVIEW

File: WTraining/app/routes.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// const controller = require('./controllers/controller');
```
Comment (emeneses4) [inline WTraining/app/routes.js]: I recommend you to remove comments

File: WTraining/app/routes.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+const { healthCheck } = require('./controllers/healthCheck');
+
+exports.init = app => {
+  app.get('/health', healthCheck);
+  // app.get('/endpoint/get/path', [], controller.methodGET);
```
Comment (emeneses4) [inline WTraining/app/routes.js]: I recommend you to remove comments

File: WTraining/app/routes.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  // app.put('/endpoint/put/path', [], controller.methodPUT);
```
Comment (emeneses4) [inline WTraining/app/routes.js]: I recommend you to remove comments

File: WTraining/app/routes.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+  // app.post('/endpoint/post/path', [], controller.methodPOST);
```
Comment (emeneses4) [inline WTraining/app/routes.js]: I recommend you to remove comments

File: WTraining/LICENSE.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 Wolox
```
Comment (marioportela1993) [inline WTraining/LICENSE.md]: Change year to 2021 :)

File: WTraining/README.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+This project is maintained by [Wolox](https://github.com/wolox) and it was written by [Wolox](http://www.wolox.com.ar).
+
+![Wolox](https://raw.githubusercontent.com/Wolox/press-kit/master/logos/logo_banner.png)
+
+## License
+
+**WTraining** is available under the MIT [license](LICENSE.md).
+
+    Copyright (c) 2019 Wolox
```
Comment (marioportela1993) [inline WTraining/README.md]: Same comment, change the year :)

File: WTraining/app/middlewares/apiInfo.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+const pjson = require('../../package.json');
```
Comment (marioportela1993) [inline WTraining/app/middlewares/apiInfo.js]: Change the name of this file to api_info.js in accordance with the document of good practices :)

File: WTraining/app/routes.js (hunk 1 above)
Comment (marioportela1993) [inline WTraining/app/routes.js]: Delete comments please

File: WTraining/package.json [hunk 8]
```diff
@@ -0,0 +1,111 @@
+{
+  "name": "WTraining",
+  "version": "0.1.0",
+  "description": "WTraining",
+  "engines": {
+    "node": "10.14.1",
```
Comment (marioportela1993) [inline WTraining/package.json]: Which version of node are you using?

**Review by marioportela1993 — CHANGES_REQUESTED** [review]
Good job, please check the comments and let me know if you have questions.

**Review by marioportela1993 — APPROVED** [review]

**Pull request merged**